cmd/osctl/cmd/kubeconfig.go

```diff
@@ -55,12 +57,21 @@ Kubeconfig will be written to PWD/kubeconfig or [local-path]/kubeconfig if speci
 			if len(args) == 1 {
 				localPath = args[0]
 			}
+			localPath = filepath.Clean(localPath)
 
-			extractTarGz(filepath.Clean(localPath), r)
+			// Drop the existing kubeconfig before writing the new one if force flag is specified.
+			if force {
+				err = os.Remove(filepath.Join(localPath, "/kubeconfig"))
```

Comment:
super nit: the extra `/` in `filepath.Join(localPath, "/kubeconfig")` isn't needed.

Reply:
fixed, hopefully ready for merge now.